### PR TITLE
Happy blocks: Update header desktop and mobile

### PR DESCRIPTION
**Pull request by renancarvalho**
## Proposed Changes

This PR updates the `happy blocks` header according to [new designs](IceKRdz5qlPYwuqzu6pBgF-fi-2272_29779), It contains mainly mobile changes, a divider added intentionally below the menu, the search new location and menu colors that now are blue.

Before:
image

image

image
image

After:

image

image

image

image

## Testing Instructions

- Pull and run this branch
- Navigate to `cd apps/happy-blocks` and run `yarn dev --sync`
- Sandbox `learncft.wordpress.com` and `wordpress.com/support` and visit the site
- Check if the designs are matching and there's no weirdness around the header. 
- Check [learn](https://learncft.wordpress.com/) and mainly [support](https://wordpress.com/support/) since it is live
- Check mobile and desktop.

I have checked RTL and looks fine but also good, a second eye.

@vinimotaa, your check here would be highly appreciated 🙇

### CODE REVIEW

**Review by escapemanuele — APPROVED** [review]
Much better now!

**Comment by renancarvalho** [comment]
After discussing with @vinimotaa, we agreed that it needs to be tested and checked before going live once it impacts `/support`. Furthermore, Vini needs to check how it would look or redesign it on the inner pages.

For now, let's keep it on hold until I find out how we can make this available for testing and Vini's output on the inner pages.

**Comment by escapemanuele** [comment]
It is working great! Only thing I noticed may be this:

This PR:
image

Figma:
image

There may be the need for some more distance between the title and the subtitle maybe!

File: apps/happy-blocks/block-library/universal-header/style.scss [hunk 1]
```diff
@@ -702,15 +718,41 @@ body.logged-in .x-dropdown {
...
+	margin-top: 24px;
+	@media ( max-width: $breakpoint-mobile ) {
+		flex-direction: column;
+		margin-top: 0;
+	}
 	h1 {
 		font-family: $recoleta-font-family;
 		font-size: 2.75rem;
-		margin-bottom: 5px;
+		line-height: 40px;
```
Comment (renancarvalho) [inline apps/happy-blocks/block-library/universal-header/style.scss:731]: The line-height is not according to Figma, we came with this value on a huddle call.

**Pull request merged**